perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -165,10 +165,13 @@ sub ward : Path : Args(2) {
     $c->stash->{stats} = $c->cobrand->get_report_stats();
 
     my @categories = $c->stash->{body}->contacts->not_deleted->search( undef, {
-        columns => [ 'id', 'category', 'extra' ],
+        columns => [ 'id', 'category', 'extra', 'body_id', 'send_method' ],
         distinct => 1,
         order_by => [ 'category' ],
     } )->all;
+
+    $c->cobrand->call_hook('munge_categories', \@categories);
```

Comment:
Lots of similar named functions, I'm having trouble keeping track. Perhaps this should be munge_reports_category_filter_list? (and munge_category_where could be munge_around_category_filter_where or somesuch)